### PR TITLE
Inital commit

### PR DESCRIPTION
**Pull request by skymandr**
A lambda for getting and inserting Level0 data.

When a new file shows up in a bucket, the lambda reacts.

1. First it sets up the postgresql environment
2. Then it downloads credentials it needs for the database
3. Then it get the file to work on
4. Finally it inserts it in the remote postgresql database

... or at least that's the idea...

Resolves https://github.com/Odin-SMR/odincal/issues/15 (maybe)

### CODE REVIEW

File: cdk.context.json
Comment (joakimmoller) [inline cdk.context.json:1]: ska den med i repot?

File: level0/handlers/handler.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        os.environ["PGSSLKEY"] = str(pg_key_path)
+
+        ssm_client: SSMClient = boto3.client("ssm")
+        db_host = ssm_client.get_parameter(
+            Name=pg_host_ssm_name,
+            WithDecryption=True,
+        )["Parameter"]["Value"]
+        db_user = ssm_client.get_parameter(
+            Name=pg_user_ssm_name,
+            WithDecryption=True,
```
Comment (joakimmoller) [inline level0/handlers/handler.py:108]: user is an cleartext parameter; "WithDecryption"?

**Review by joakimmoller — APPROVED** [review]
:shipit:

File: level0/handlers/handler.py (hunk 1 above)
Comment (skymandr) [reply level0/handlers/handler.py:108]: copy-pejst...

File: cdk.context.json
Comment (skymandr) [reply cdk.context.json:1]: Jag vet faktiskt inte. Den kanske automatgenereras vid synth..?

**Review by joakimmoller — APPROVED** [review]
Looks like a good structure.

File: level0/notify_l1_handler/handler.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    return short_hash
+
+
+def handler(event: Event, context: Context) -> dict[str, int]:
+
+    state_machine_arn = find_arn()
+    if state_machine_arn is None:
+        raise StateMachineError(
+            f"no matching state machine {STATE_MACHINE_NAME} found"
+        )
```
[the remark this answers is not in the document]
Comment (skymandr) [reply level0/notify_l1_handler/handler.py:47]: Yes, forgot to remove this. Well spotted.

Comment (skymandr) [inline level0/notify_l1_handler/handler.py:47]:
```suggestion
```

**Pull request merged**